scripts/Units/armsnipe.bos

```diff
@@ -7,7 +7,7 @@ piece  flare, torso, head, pelvis, rthigh, rleg, lleg, lthigh,
        ruparm, luparm, lloarm, lhand, rloarm, gun1, gun2,
        barrel1, barrel2, laser, rhand, rfoot, lfoot, aimy1;
 
-static-var  GunOpen, bMoving, Static_Var_3, Static_Var_4, restore_delay, wpn1_lasthead, animSpeed, maxSpeed,
+static-var  GunOpen, bMoving, Static_Var_3, Static_Var_4, restore_delay, laser_hide, wpn1_lasthead, animSpeed, maxSpeed,
```

Comment:
While you're here, can you figure out what `Static_Var_3` and `Static_Var_4` are, and give them meaningful names?

Reply:
Static_Var_# both seem to be 'state trackers' related to animation state functions. However, there are no meaningful checks to them, so if they were removed there would be no apparent difference.

Reply:
Static_Var_1 is changed to GunOpen. 3 has always been static variable 1's puppet. Static_Var_4 has been this variable that exists to know if the walk thread should be active or not, and its purpose is to lag aiming when standing still. Since GunOpen exists and var4 seems useless, I'll just delete them.